### ISSUE TITLE
Add Configuration in a file instead of interactive first time setup

### ISSUE THREAD

**Issue by Appyx** [Taxel/PlexTraktSync#68]
Hi,
I really like this project :)

I'm running this in a docker container, so that i can start the container when needed.
The only problem is that I can't figure out how to do the initial setup without the interactive part.

So my workaround is to do the first setup manually and then only copy the files to the image during the docker build.

I would open a pull request for the docker setup...but It needs manual work to get there.

TL;DR: Please add a config file with a non interactive setup in order to dockerize the project.

**Comment by twolaw** [comment]
Hi,
I'm running it in a docker container too.
The solution is to do the [first run interactively](https://github.com/twolaw/PlexTraktSync/tree/docker#docker).

**Comment by Appyx** [comment]
Hi @twolaw,

Thanks for sharing your solution. It looks way more advanced than mine 👍 

Maybe I will look into packaging the whole thing into a docker image on docker hub.

**Comment by twolaw** [comment]
Docker Hub already have 2 repositories for plextraktsync on linux/amd64.
I will push my image built for linux/arm if it can help.
